shaquille/stocks_challenge.py

```diff
@@ -0,0 +1,71 @@
+print ("Challenge 3.2: Playing with the stock market")
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+amazon = 3000
+apple = 100
+facebook = 250
+google = 1400
+microsoft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# TODO: Write code to ask the client his name and save it to a variable.
+Name= input("what is your name? ")
+# TODO: Write code to ask the client his savings and save it to another variable.
+Savings= input("What is your savings?")
+# TODO: Write code to ask the client the stock he is interested in and save it to another variable, as shown below.
+stock = input("Which stock are you interested in? Type 'amzn' for Amazon, 'aapl' for Apple, 'fb' for Facebook, 'goog' for Google and 'msft' for Microsoft.")
+
+print("Challenge 3.2.2: Perform user-specific calculations")
+# TODO: You have all 3 user inputs stores in variables. Based on that, write conditional (if-elif-else) statements to find out the number of stocks of the company that can be purchased with the savings amount.
+
+amzn=amazon
+aapl=apple
+fb=facebook
+goog=google
+msft=microsoft
+
+number_of_stocks= 0
+
+if stock == amzn:
+    number_of_stocks=(Savings/amazon)
+elif number_of_stocks>0:
+    print(f"Congratulations {Name} you have {Savings} of savings and you can buy {number_of_stocks} Amazon stocks ")
+elif number_of_stocks<0:
+    print("Sorry, you can not buy Amazon stocks with the {savings}")
+elif stock== aapl:
```

Comment:
I think you need 'aapl' (and the subsequent ones) in quotes. Right now you're checking if stock is equal to the aapl variable which you set to apple which is 100